python-requirements.txt

```diff
@@ -0,0 +1,7 @@
+coloredlogs==15.0.1
+pyaml==19.12.0
```

Comment:
Do we want this specific version, or anything >= 19?

Reply:
We probably want to use `~=` instead of `==`. That will request a version "compatible with" the specified version. So it won't go up a major version, but will allow for bugfix or minor updates.